syntaxes/mnscript.tmLanguage.json

```diff
@@ -87,6 +149,36 @@
 					"match": ","
 				}
 			]
+		},
+		"storage":{
+			"patterns": [
+				{
+					"name":"storage.type.function.mnscript",
+					"match":"function"
+				},
+				{
+					"name":"storage.type.class.mnscript",
+					"match":"class"
+				},
+				{
+					"name":"storage.type.mnscript",
+					"match": "string|number|bool|string\\[\\]|number\\[\\]|bool\\[\\]",
+					"include": "#variables"
+				}
+			]
+		},
+		"variables":{
+			"begin": "string|number|bool|string\\[\\]|number\\[\\]|bool\\[\\]",
+			"end": ";",
+			"beginCaptures": {
+				"0":{ "name":"storage.type.mnscript"}
+			},
+			"patterns": [
+				{
+					"match": "(?<=\\=\\s+)[a-zA-Z\\_0-9]+|[a-zA-Z\\_0-9]+(?=;)|[a-zA-Z\\_0-9]+\\s*(?=\\=)|[a-zA-Z\\_0-9]+\\s*(?!\\))|(?<!\\()[a-zA-Z\\_0-9]|(?<!\\{)[a-zA-Z\\_0-9]",
+					"name": "variable.name.mnscript"
+				}
+			]			
```

Comment:
I don't recall exactly if the match and begin are strictly necessary, and this bit probably needs a refactor anyway, but that's for another day when the actual language server portion gets implemented.

Respond to the first sentence, this isn't something that *needs* to be resolved for merging. Just laying out my thoughts in text.

Reply:
This does need to be refactored, it's a mess!
I toyed around with removing the begin and end statement, but it comes back to the issue of not highlighting keywords like string correctly, which is why I ended up doing the weird capture to begin with.
Will definitely need to come back and look at this at some point, for now just going to leave it in.

Reply:
No problem.